### ISSUE TITLE
Window sizing

### ISSUE THREAD

**Issue by Ads20000** [PhoenicisOrg/POL-POM-5#1051]
![screenshot from 2017-08-23 22-55-18](https://user-images.githubusercontent.com/3178485/29640419-ceee5944-8856-11e7-924f-6e1a1adddc6f.png)

I'm not sure about your sizing of the right panel here when the Phoenicis window is made smaller. Here the text gets cut off in the right-hand panel, would it not just be better to auto-resize so the text wraps? Then you should probably prevent the user from making the window any smaller when it starts hitting the bottom image (or allow the user to go a bit further by auto-resizing the bottom image).

Also, you can see that the entries in the top panel are cut off - I like how you've dealt with this by enabling the scroll wheel to scroll and providing a drop-down button, you may want to have small left-right arrow buttons like Firefox does when lots of tabs are open.

![screenshot from 2017-08-23 23-04-55](https://user-images.githubusercontent.com/3178485/29640593-812ee902-8857-11e7-806a-5ebd4f616e70.png)

But you need to set a minimum size because at the moment you can make ridiculous windows like

![screenshot from 2017-08-23 23-04-15](https://user-images.githubusercontent.com/3178485/29690659-be4a7414-891f-11e7-8173-d67064175ca0.png)
![screenshot from 2017-08-23 23-04-23](https://user-images.githubusercontent.com/3178485/29690658-bdb26fb6-891f-11e7-8467-8f839bdca50b.png)
![screenshot from 2017-08-23 23-04-28](https://user-images.githubusercontent.com/3178485/29690669-c6a5433c-891f-11e7-9423-9f1406f367c9.png)

Most programs don't allow the user to do this. And it looks very bad that when resizing too low you just end up cutting off most of the UI (at the very least, the user should be able to scroll across the UI elements to be able to access all of the UI elements) but

**Comment by plata** [comment]
I disagree that we should have a minimum window size. Yes, it doesn't make sense to have a window which is that small but there's not reason to enforce a minimum size. It's also not true, that other applications do that.
For example, Firefox:
![firefox](https://user-images.githubusercontent.com/3973260/29707995-8ad25d50-8987-11e7-9b0a-abedb1edd6a3.png)

On the contrary, I have seen applications which enforce a minimum window size which suddenly got unusable e.g. on lower resolutions because you couldn't access all buttons anymore.

The handling of many tabs is just the default of JavaFX. I don't know if there's a simple way to implement something like Firefox.

You're definitely right about the text in the details panel. I will open a new issue to track this (it's always difficult to have several topics in one issue).

**Comment by plata** [comment]
@Ads20000 I think we can close this, right?

**Comment by Ads20000** [comment]
Yes I guess. Personally I prefer it when windows prevent users from making the app unusable but I guess you're not going to change your mind because I don't think I have any new arguments...although in the Firefox window there you can still access everything!? Whereas in Phoenicis you can go to literally a 1x1 pixel window, which is bizarre.

**Comment by plata** [comment]
I don't think Firefox is really usable like this (you can't even see the webpage at all). If you like, I can set some arbitrary minimum size like 100x100 to avoid 1x1 windows. I just don't want to force a minimum size where Phoenicis is really usable (like 800x600) because you can never know what you cause with that like I explained above.
